deploy/rbd/kubernetes/v1.13/csi-rbdplugin-provisioner.yaml

```diff
@@ -111,6 +111,26 @@ spec:
               mountPath: /etc/ceph-csi-config/
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
+        - name: liveness-prometheus
+          image: quay.io/cephcsi/cephcsi:canary
+          args:
+            - "--type=liveness"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--livenessport=8080"
```

Comment:
just to clarify you will be using host port 8080 for rbd and 8081 for cephfs right?

do we need to enable firewall rules to listen on these ports?

Reply:
yes the defualt is 8080 for rbd, 8081 for cephfs. 

If multiple ceph clusters where deployed with multiple instances of csi this would change for the additional sets of rbd and cephfs pods (ex 8082, 8083...)

ill put this in the metrics doc as well as a mention of opening the firewall if needed.